docs/modules/integrate/pages/integrate-with-feast.adoc

```diff
@@ -0,0 +1,119 @@
+= Integrate with Feast
+:page-enterprise: true
+:description: pass:q[Feast (**Fea**ture **St**ore) is a customizable operational data system, which uses your existing infratstructure to manage and serve machine learning features to real-time models. When integrated with Hazelcast, you can benefit from an online store that supports materializing feature values in a running Hazelcast cluster.]
+
+{description}
+
+This approach unlocks the following to power your Feast real-world machine learning (ML) requirements:
+
+* One-to-one mapping of each feature view to a specific IMap data structure
++
+This means that every feature view corresponds to an IMap in the Hazelcast cluster, and the entries in that IMap correspond to features of entitites. Each feature value is stored separately, and can be retrieved individually.
```

Comment:
This paragraph sounds like the user is able to use their pre-existing IMaps and the entries in those with Feast. That's not the case. Feast creates a new IMap for each feature, and the user has no control over the entries in those IMaps.